.gitignore

```diff
@@ -8,3 +8,6 @@ iSpiEFP/out/production/iSpiEFP_GUI
 /iSpiEFP/w6b2_energy.out
 /iSpiEFP/w6b2_md.out
 /iSpiEFP/w6b2_pairwise.out
+/target/
```

Comment:
@rderue Can you take a look at this merge conflict. I don't know how you restructured the repo.